### PR TITLE
Add documentation for the Hash MSBuild task

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by ktoliver** [comment]
Email sent to @ghogen and @jillre

File: docs/msbuild/hash-task [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+|`IgnoreCase`|Optional `Boolean` parameter.<br /><br />When true, will generate a case-insensitive hash.|
+
+## Example
+
+The following example uses the `Hash` task to determine and print the checksum of the `HashItem` items.
+
+```xml
+<Project>
+  <ItemGroup>
+    <HashItem Include="A />
```
Comment (ghogen) [inline docs/msbuild/hash-task]:
```suggestion
    <HashItem Include="A" />
```

File: docs/msbuild/hash-task [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <HashItem Include="B />
```
Comment (ghogen) [inline docs/msbuild/hash-task]: Presumably there should be closing quotes in these
```suggestion
    <HashItem Include="B" />
```

**Comment by ghogen** [comment]
@pranavkm Once this is finalized here, we'll actually take it into the private repo to merge it there. (There's some policies that won't allow us to accept PRs from internal employees in the public repo - so in future, use https://github.com/MicrosoftDocs/visualstudio-docs-pr.)

**Comment by pranavkm** [comment]
I've updated the content based on your suggestion.

> Once this is finalized here, we'll actually take it into the private repo to merge it there.

👍

**Pull request closed without merging**